_test/test_sqw_class/test_dnd_copy.m

```diff
@@ -0,0 +1,41 @@
+classdef test_dnd_copy < TestCase
+
+    properties
+        sqw_file_1d_name = 'dnd_2d.sqw';
+        sqw_files_path = '../test_sqw_file/';
+
```

Comment:
why are these paths being named sqw_? It is giving the impression they are paths to sqw objects. Is dnd_ preferable?

Reply:
cut+paste error, fixed now.